Contributors.md

```diff
@@ -11,5 +11,6 @@
     - Ivan Subotic <https://github.com/subotic>
     - Loïc Jaouen <https://github.com/loicjaouen>
     - Sepideh Alassi <https://github.com/SepidehAlassi>
-    - André Kilchenmann <https://github.com/kilchenmann>
     - Stefan Münnich <https://github.com/musicEnfanthen>
+    - André Kilchenmann <https://github.com/kilchenmann>
+    - <https://github.com/janCstoffregen>
```

Comment:
Shouldn't that be "Jan Stoffregen <https://github.com/janCstoffregen>"`

Reply:
It can't be, because he hasn't put his name in his GitHub accont.